client/app/src/js/services/User.js

```diff
@@ -1,13 +1,12 @@
 function User() {
-
-  var user = {};
+  var user = null;
```

Comment:
Since we're not Wiki'ing the hell out of front end yet, we should add some notes about what `User.js` does and why we have it around.